templates/config/70_nodepools.yaml.j2

```diff
@@ -0,0 +1,38 @@
+---
+nodepools:
+  abstract:
+    mech-ansible:
+      mechanism:
+        type: "ansible"
+        ansible:
+          topdir: "/home/duffy/src/duffy-ansible"
```

Comment:
Let's make this a variable and `ansible-host` will be inherited as role and that's not where ansible workspace (including playbooks/inventory/ansible.cfg) will be

Reply:
Oversight, oops. :wink: